sdk/parents/azure-client-sdk-parent/pom.xml

```diff
@@ -1470,7 +1470,7 @@
     <module>../../core/azure-core-tracing-opentelemetry</module>
     <module>../../cosmos/azure-cosmos</module>
     <module>../../cosmos/azure-cosmos-benchmark</module> <!-- CosmosSkip - Benchmark will go away, this is temporary -->
-    <module>../../cosmos/azure-cosmos-examples</module> <!-- CosmosSkip - Examples will go away, this is temporary -->
+    <module>../../cosmos/azure-cosmos-examples</module> <!-- CosmosSkip - Benchmark will go away, this is temporary -->
```

Comment:
nit: This change is neither correct nor necessary, please undo the changes to this file and remove from this PR.

Reply:
Ah cut and paste error. I had originally removed those module lines altogether but put them back in at your request but didn't adjust the comment.